### ISSUE TITLE
Question about Olthof (2017) and searchability in general

### ISSUE THREAD

**Issue by AleksandrsBerdicevskis** [CHIELDOnline/CHIELD#196]
If, as a user, I want to see all graphs that have to do with language complexity, I'd like to see this one, too, but if I search for "complexity", I won't. At the same time, I see that it would probably be wrong to change the variable label to smth like "complexity: transparency", since that's not what the author is saying.
Is there a good solution to such problems? Maybe add some keywords to the graphs/documents (without linking them to specific variables)?

Document: [olthof2017transparency](https://chield.excd.org/document.html?key=olthof2017transparency)
Contributors: @sinnemaki

**Comment by seannyD** [comment]
I see what you're saying.  You could add "complexity" into the "topic" column of the link. Another way (though not exactly what you want) is to use the Explore page.  Add "language contact vs. change", then click the "expand" button.  Then you'll see some links with complexity.

**Comment by AleksandrsBerdicevskis** [comment]
Thanks! Right, I forgot about the "topic". What do you think, Kaius, should it be added as a topic here?

**Comment by sinnemaki** [comment]
I agree with Sasha that the author does not equate transparency with complexity, so maybe changing the label is not the optimal solution. But from a more general perspective Olthoff's paper as well as the notion of transparency are clearly related to work on language complexity. Adding complexity as a topic would work perfectly!
